src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/masonry-decorator/masonry-view.js

```diff
@@ -182,7 +182,7 @@ define([
             this.initializeMasonryGrid();
             this.bindGeneralDomEvents();
 
-            this.renderRecords(data.embedded);
+            this.renderRecords(data.embedded, true);
```

Comment:
It has to be done this way, otherwise the order is reversed. That logic is actually quite wonky :confused: